web/index_test.go

```diff
@@ -0,0 +1,29 @@
+package web_test
```

Comment:
Is it Go convention to put tests in `<pk being tested>_test`?

Reply:
Seems that way